sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Models/EncryptResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public class EncryptResult
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public byte[] Ciphertext { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public byte[] AuthenticationTag { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public object Algorithm { get; set; }
```

Comment:
Should be type EncryptionAlgorithm

Reply:
good catch, as a matter of fact, I wanted to update this, but by that time my enum was not ready. Will update it.